logger/proto.go

```diff
@@ -49,7 +49,7 @@ func (p protoMarshaller) MarshalLogObject(e zapcore.ObjectEncoder) error {
 		v := p.m.Get(f)
 
 		if proto.HasExtension(f.Options(), logger.E_Redact) {
-			e.AddString(k, "<redacted>")
+			e.AddString(k, fmt.Sprintf("<%s>", f.TextName()))
```

Comment:
the name is already the field key in the log and replacing the value with the text "redacted" makes it clear why it's been omitted to readers unfamiliar with the message schema.

Reply:
The trade off is consistency with the existing logic. I have no strong opinion either way. @frostbyte73 what is your opinion here since I believe you wrote the existing redaction code?